client/lib/domains/whois/test/assembler.js

```diff
@@ -6,105 +6,144 @@ const expect = require( 'chai' ).expect;
 /**
  * Internal dependencies
  */
-const whoisAssembler = require( './../assembler' );
+const whoisAssembler = require( './../assembler' ),
+	{ whoisType } = require( './../constants' );
+
+const FIRST_NAME = 'First-name',
+	LAST_NAME = 'Last-name',
```

Comment:
I don't think there is an explicit preference in the JS guidelines I've noticed that most new commits are using multiple const (or let) statements over combined statements.
This has a couple of benefits as mentioned in the [AirBnB guidelines](https://github.com/airbnb/javascript#variables--one-const)
Just a suggestion though, since there's no mention of this in Calypso's guidelines - feel free to ignore :)